test/new-e2e/pkg/runner/local_profile.go

```diff
@@ -114,3 +115,21 @@ func (p localProfile) NamePrefix() string {
 func (p localProfile) AllowDevMode() bool {
 	return true
 }
+
+// GetOutputDir extends baseProfile.GetOutputDir to create a symlink to the latest run
+func (p localProfile) GetOutputDir() (string, error) {
+	outDir, err := p.baseProfile.GetOutputDir()
+	if err != nil {
+		return "", err
+	}
+
+	// Create a symlink to the latest run for user convenience
+	latestLink := filepath.Join(filepath.Dir(outDir), "latest")
+	_ = os.Remove(latestLink)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Can this fail ?

Reply:
updated thanks!